putio.py

```diff
@@ -9,6 +9,15 @@
 import requests
 import iso8601
 
+KB = 1024
+MB = 1024 * KB
+
+# Downloads are broken up into ranges of this size
+RANGE_SIZE = 10 * MB
+
+# Downloads are written and flushed after a chunk of this size
+CHUNK_SIZE = 8 * KB
```

Comment:
How did you choose 8K? I guess a bigger value such as 256K may increase write performance or reduce load on the cpu/disk? Did you do any tests?